packages/wasm/package.json

```diff
@@ -1,27 +1,32 @@
 {
   "name": "@penumbra-zone/wasm",
   "version": "3.0.0",
-  "private": true,
   "license": "MIT",
   "type": "module",
   "scripts": {
     "clean": "rm -rfv dist wasm ; cargo clean --manifest-path ./crate/Cargo.toml",
     "dev": "cargo watch -C ./crate --postpone -- pnpm turbo run compile",
     "compile": "cd crate ; wasm-pack build --no-pack --target bundler --out-name index --out-dir ../wasm",
-    "build": "tsc",
+    "prepack": "touch ./wasm/.npmignore",
+    "build": "tsc -noEmit",
     "lint": "eslint \"src/*.ts*\"",
     "lint:rust": "cd crate && cargo clippy -- -D warnings && cargo clippy --tests -- -D warnings",
     "format-check:rust": "cd crate && cargo fmt --all -- --check",
     "test": "vitest run",
     "test:rust": "cd crate ; wasm-pack test --headless --firefox -- --test build --target wasm32-unknown-unknown --release --features 'mock-database'"
   },
+  "exports": {
+    "./*": "./src/*.ts"
+  },
   "dependencies": {
-    "@penumbra-zone/keys": "workspace:*",
     "@penumbra-zone/types": "workspace:*",
     "zod": "^3.22.4"
   },
   "devDependencies": {
     "@penumbra-zone/bech32": "workspace:*",
     "fake-indexeddb": "^5.0.2"
+  },
+  "optionalDependencies": {
+    "@penumbra-zone/keys": "workspace:*"
```

Comment:
Interesting, never heard of this category of deps

Reply:
Is it required to be a dynamic import like so? https://github.com/turbocrime/nextjs-penumbra-wasm-example/blob/360201378c47cc217a704ceaf8b32bce72e120a9/app/components/ComponentWithWasm.tsx#L6

Reply:
this is the way i could get it to work with next. will try more today

Reply:
note that isn't the keys package import that's a submodule that happens to be named keys